src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2

```diff
@@ -0,0 +1,306 @@
+The framing of SLCs is not always consistent along a path, so bursts may be included at the top of one SLC for one
+pass and the bottom of the next SLC for a different pass. This makes it more difficult to find and process all the
+acquisitions available for a specific burst over time. When searching through time at the burst level, you can be
+assured that your stack includes all the acquisitions for that burst footprint.
+
+Note that the imaging surface will vary depending on the sensor wavelength. Because Sentinel-1 is a C-band sensor,
+the waves will not penetrate very deeply into vegetation. The imagery likely represents the top of the canopy in
+densely vegetated areas rather than the actual terrain. In addition, vegetated areas tend to have low coherence,
+because plants can grow or move from one acquisition to the next. Use caution when generating interferograms for
+areas with extensive/dense vegetation cover.
+
+A digital elevation model is required for processing InSAR. The DEM used for this particular product is the
+Copernicus GLO-30 Public DEM, which has a native resolution of 1 arc second (about 30 meters). This DEM is a
+global Digital Surface Model (DSM) derived from the WorldDEM. The WorldDEM is based on radar satellite data
+acquired by the TanDEM-X mission, which was funded by the German Aerospace Center (DLR) and Airbus Defence and Space,
+and edited to flatten water bodies, provide consistent flow of rivers, and apply corrections to shore/coastlines and
+special features. For an overview of the dataset, visit
+https://spacedata.copernicus.eu/collections/copernicus-digital-elevation-model.
+
+***************
+# Product Contents #
+
+The files generated in this process include:
+
+1. Wrapped Interferogram (GeoTIFF)
+2. Unwrapped Interferogram (GeoTIFF, PNG image)
+3. Coherence Map (GeoTIFF)
+4. Connected Components (GeoTIFF)
+5. Parameter Documentation (Text File)
+6. Look Vector Maps (GeoTIFFs)
+7. DEM used to process the data (GeoTIFF)
+8. README.md.txt (Text File)
+
+
+
+*See below for detailed descriptions of each of the product files.*
+
+----------------
+## 1. Wrapped Interferogram
+
+The wrapped interferogram is output in GeoTIFF file. Because the fringes are on a wrapped 2-pi scale rather than representing continuous phase difference values, a wrapped interferogram tends to be most useful as a color visualization. This product displays deformation in multiples of half of the sensor wavelength, so areas with fringes that are very close together indicate greater amounts of deformation in the line-of-sight direction from the sensor. Each 2-pi cycle is equivalent to about 2.8 cm in line-of-sight displacement (half the wavelength of the Sentinel-1 sensor, which is about 5.6 cm).
+
+The wrapped interferogram image is generated from the interferogram phase prior to unwrapping. The PNG and KMZ files are tagged with _color_phase, and are 2048 pixels wide. The optional GeoTIFF is tagged with _wrapped_phase.tif
```

Comment:
Are we including PNG and KMZ files for the wrapped phase? Is the GeoTIFF version included and/or optional?